app/controllers/api/enterprises_controller.rb

```diff
@@ -77,5 +77,12 @@ def override_sells
     def override_visible
       params[:enterprise][:visible] = false
     end
+
+    # Allows API access without a logged in user for actions in this controller.
+    # Actions that require authentication should all use #authorize!
+    # @current_api_user will now initialize an empty Spree::User unless one is present.
```

Comment:
IMO it's worth explaining why we needed this and the tradeoff we made. That context will be invaluable the next time we end up here, don't you think?

Reply:
@sauloperez @Matt-Yorkley updated the comment - are you already happy with it?